v4/export/writer_serial_test.go

```diff
@@ -67,15 +24,19 @@ func (s *testWriteSuite) TestWriteMeta(c *C) {
 	writer := storage.NewBufferWriter()
 
 	err := WriteMeta(tcontext.Background(), meta, writer)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
+
 	expected := "/*!40103 SET TIME_ZONE='+00:00' */;\n" +
 		"CREATE TABLE `t1` (\n" +
 		"  `a` int(11) DEFAULT NULL\n" +
 		") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;\n"
-	c.Assert(writer.String(), Equals, expected)
+	require.Equal(t, expected, writer.String())
 }
 
-func (s *testWriteSuite) TestWriteInsert(c *C) {
+func TestWriteInsert(t *testing.T) {
+	cfg, clean := createMockConfig(t)
+	defer clean()
```

Comment:
can we use [`testify/suite`](https://pkg.go.dev/github.com/stretchr/testify@v1.7.0/suite) for these instead of inlining the setup/teardown code to every test function 🤷

Reply:
Although it is fine for serial tests, I tend to avoid testify/suite.

See also this comment https://github.com/pingcap/tidb/pull/26082#issuecomment-877230972 and my [blog](https://mp.weixin.qq.com/s/8q776_5ZZDwA1IX_m-q1CQ) (in Chinese) about it.